cmd/blockchaincmd/deploy.go

```diff
@@ -607,34 +606,16 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 	}
 
 	if !sidecar.NotSOV {
```

Comment:
nit: not sure if it's in the scope of the PR but why we have an inverse logic here, should `!sidecar.NotSOV` be just something like sidecar.SOV?